concordium-node/src/stats_export_service.rs

```diff
@@ -467,6 +470,16 @@ impl StatsExportService {
         let last_throughput_measurement_received_bytes = AtomicU64::new(0);
         let avg_bps_in = AtomicU64::new(0);
         let avg_bps_out = AtomicU64::new(0);
+        let peer_bucket_size = IntGaugeVec::new(
+            Opts::new(
+                "peer_bucket_size",
+                "The number of peers that recently connected to the bootstrapper labelled by the \
+                 number of the bucket in which they are contained",
```

Comment:
I just realized that this help text is not the best. It talks about bootstrappers but actually the buckets are there for normal nodes as well. Maybe just replace bootstrapper with node

Reply:
Good point!